### DISCUSSION TITLE
SILENT and VERYSILENT aren't working in 18.2

### DISCUSSION THREAD

**Discussion by jovball** [JanDeDobbeleer/oh-my-posh#4113] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I'm using a PowerShell script to install and configure oh-my-posh for the current user. I would like the install to be completely silent. If I use either the /SILENT switch or /VERYSILENT switch, the installer loads the opening dialog asking the user to select the install mode.

`
.\install-amd64.exe /LOADINF="oh-my-posh.inf" /VERYSILENT
`

### Theme

N/A

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
N/A
```

**Comment by JanDeDobbeleer** [comment]
@jovball nothing changed there so I'll check.

**Comment by JanDeDobbeleer** [comment]
@jovball this works for me without fail, so either the `/LOADINF="oh-my-posh.inf"` part is what's causing that on your end or something else is interfering. But I can download any installer and it will respect those flags (they are built-in by inno, so we don't even control those).
